modules/media/jsx/mediaIndex.js

```diff
@@ -56,6 +74,7 @@ class MediaIndex extends Component {
    * @return {*} a formated table cell for a given column
    */
   formatColumn(column, cell, row) {
+    cell = this.mapContainerColumns(column, cell);
```

Comment:
```suggestion
    cell = this.mapColumn(column, cell);
```

Reply:
With the change suggested above, the module loads properly and the CSV downloaded has the proper mapping for site.

Reply:
Super strange, I swear I changed and comitted that!